Sources/SwiftCrossUI/App.swift

```diff
@@ -52,6 +78,8 @@ private enum SwiftBundlerMetadataError: LocalizedError {
 
 extension App {
     /// Metadata loaded at app start up.
+    ///
+    /// This is accessible from within ``init()``.
```

Comment:
Rewording suggestion to make it clear what accessible means (given that it's accessible from basically everywhere):

```
This will contain the app's metadata, if present, by the time ``init()`` gets called.
```